packages/console/src/pages/Users/index.tsx

```diff
@@ -75,83 +77,88 @@ const Users = () => {
           />
         </Modal>
       </div>
-      <div className={styles.filter}>
-        <Search
-          defaultValue={keyword}
-          isClearable={Boolean(keyword)}
-          onSearch={(value) => {
-            setQuery(value ? { search: value } : {});
-          }}
-          onClearSearch={() => {
-            setQuery({});
-          }}
-        />
-      </div>
-      <div className={classNames(styles.tableContainer, tableStyles.scrollable)}>
-        <table className={classNames(!data && tableStyles.empty)}>
-          <colgroup>
-            <col className={styles.userName} />
-            <col />
-            <col />
-          </colgroup>
-          <thead>
-            <tr>
-              <th>{t('users.user_name')}</th>
-              <th>{t('users.application_name')}</th>
-              <th>{t('users.latest_sign_in')}</th>
-            </tr>
-          </thead>
-          <tbody>
-            {!data && error && (
-              <TableError
-                columns={3}
-                content={error.body?.message ?? error.message}
-                onRetry={async () => mutate(undefined, true)}
-              />
-            )}
-            {isLoading && <TableLoading columns={3} />}
-            {users?.length === 0 && (
-              <TableEmpty columns={3}>
-                <Button
-                  title="users.create"
-                  type="outline"
-                  onClick={() => {
-                    setIsCreateFormOpen(true);
-                  }}
-                />
-              </TableEmpty>
-            )}
-            {users?.map(({ id, name, avatar, lastSignInAt, applicationId }) => (
-              <tr
-                key={id}
-                className={tableStyles.clickable}
-                onClick={() => {
-                  navigate(`/users/${id}`);
-                }}
-              >
-                <td>
-                  <ItemPreview
-                    title={name ?? t('users.unnamed')}
-                    subtitle={id}
-                    icon={
-                      <img
-                        alt="avatar"
-                        className={styles.avatar}
-                        src={avatar ?? generateAvatarPlaceHolderById(id)}
-                      />
-                    }
-                    to={`/users/${id}`}
-                    size="compact"
+
+      <div className={classNames(resourcesStyles.table)}>
+        <div className={tableStyles.scrollable}>
+          <table className={classNames(styles.table, !data && tableStyles.empty)}>
+            <colgroup>
+              <col className={styles.userName} />
+              <col />
+              <col />
+            </colgroup>
+            <thead>
+              <tr>
+                <th colSpan={userTableColumn} className={styles.filter}>
+                  <Search
```

Comment:
Per offline discussion, the filter should not be inside `table`

Reply:
Updated.